### DISCUSSION TITLE
Browser Extension

### DISCUSSION THREAD

**Discussion by not-matthias** [solidtime-io/solidtime#236] [Feature requests]
Hey, I really love solidtime. It would be nice to have a browser extension to start/stop time tracking. Lots of other tools (e.g. Toggl) have one.

**Comment by korridor** [comment]
solidtime now has a [Chrome](https://chromewebstore.google.com/detail/solidtime/hpanifeankiobmgbemnhjmhpjeebdhdd) and [Firefox](https://addons.mozilla.org/en-US/firefox/addon/solidtime/) extension.
